### ISSUE TITLE
Esophageal fibrosis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10043]
**Preferred term label:**
Esophageal fibrosis

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Plummer-Vinson syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000065
- label: Esophageal fibrosis
- parents: [Abnormal esophagus morphology (HP:0002031)](https://hpo.jax.org/app/browse/term/HP:0002031)
- pmid: PMID:18161745|PMID:33230074
- def: Fibrosis located in the subepithelial portion of the esophagus. Fibrosisis is defined by the overgrowth, hardening, and/or scarring of various tissues and is attributed to excess deposition of extracellular matrix components including collagen.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10043
- synonyms:
